backends/python/server/text_embeddings_server/models/__init__.py

```diff
@@ -66,10 +67,16 @@ def get_model(model_path: Path, dtype: Optional[str]):
         ):
             return FlashBert(model_path, device, dtype)
         else:
-            return DefaultModel(model_path, device, dtype, trust_remote=TRUST_REMOTE_CODE)
+            if config.architectures[0].endswith("Classification"):
```

Comment:
I think this check is a bit hacky and not very robust. Maybe a better solution is to check whether `config.model_type` is in https://github.com/huggingface/transformers/blob/165116bc145dcc186fa287e624b28a9ab3a79955/src/transformers/models/auto/modeling_auto.py#L847 ?

Reply:
I was referring to the Rust backend implementations here:[L368](https://github.com/huggingface/text-embeddings-inference/blob/main/router/src/lib.rs#L368). But I think your advice seems better :)